### PR TITLE
Check for the "jpeg" and "png" support of GD

### LINKED ISSUE #3568: Detect GD's jpeg support in SystemCheckPage

### PR DESCRIPTION
**Pull request by dtdesign**
Closes #3568

### CODE REVIEW

File: wcfsetup/install/files/acp/templates/systemCheck.tpl [hunk 1]
```diff
@@ -104,6 +104,25 @@
...
+	
+	<dl{if !$results[php][gd][result]} class="formError"{/if}>
+		<dt>{lang}wcf.acp.systemCheck.php.gd{/lang}</dt>
+		<dd>
+			{if $results[php][gd][result]}
+				{@$statusOk} {lang}wcf.acp.systemCheck.pass{/lang}
+			{else}
+				<ul class="nativeList">
+					{if !$results[php][gd][jpeg]}
+						<li>{@$statusInsufficient} <kbd>jpeg</kbd></li>
```
Comment (Gravatronics) [inline wcfsetup/install/files/acp/templates/systemCheck.tpl:116]: Why are you using `<kdb>` here but quotation marks in `wcf.acp.systemCheck.php.gd.description`? (Same applies to the next `<li>`.)

**Review by Gravatronics — COMMENTED** [review]
By manually changing the code in the PHP file to test different scenarios, I found that there is no visual difference between `gd_info()` not being available and it being available but support for both `jpeg` and `png` being missing (not sure if this is a relevant case).

File: wcfsetup/install/files/acp/templates/systemCheck.tpl (hunk 1 above)
Comment (dtdesign) [reply wcfsetup/install/files/acp/templates/systemCheck.tpl:116]: I kept it consistent to the list of missing PHP extensions.

**Comment by dtdesign** [comment]
I'm not overly concerned by this, it will report the missing `gd` extension anyway, so this is a bit redundant at worst.

**Pull request merged**